iree/hal/cuda/cts/CMakeLists.txt

```diff
@@ -27,3 +27,28 @@ iree_hal_cts_test_suite(
     "semaphore_submission"
     "semaphore"
 )
+
+# Variant test suite using graph command buffers (--cuda_use_streams=0)
+iree_hal_cts_test_suite(
+  DRIVER_NAME
+    cuda
+  VARIANT_SUFFIX
+    _graph
```

Comment:
would be nice to have this joined with _ in the rule itself - reads better here ("variant is `graph`")

Reply:
👍 good idea, moved the suffix underscore joining into the function.